addons/br/nfse/extensions.go

```diff
@@ -12,14 +12,33 @@ import (
 // of these extensions are common, they can be moved to the regime or to a
 // shared addon.
 const (
+	ExtKeyCNAE            = "br-nfse-cnae"
 	ExtKeyFiscalIncentive = "br-nfse-fiscal-incentive"
+	ExtKeyISSLiability    = "br-nfse-iss-liability"
 	ExtKeyMunicipality    = "br-nfse-municipality"
 	ExtKeyService         = "br-nfse-service"
 	ExtKeySimplesNacional = "br-nfse-simples-nacional"
```

Comment:
With so many extensions now, "simples nacional" feels even more out of place... would `br-nfse-reu` make sense? I've been playing with ChatGPT, it stands for "Regime Especial Unificado". At least with a Code it forces you to read the definition as opposed being confused by a foreign language....

Reply:
Addressed by 44a3b9ee9e6c524f59385ad1886f05340421f7e7